### PR TITLE
feat(master): create puppet master

### PR DESCRIPTION
**Pull request by rbeuque74**
Creates a puppet master

References #6

### CODE REVIEW

File: ppaas/master.py [hunk 1]
```diff
@@ -107,6 +107,48 @@ def reload_data(self, refreshed_datas=None):
...
+        :type name: str
+        :param source: Source git repository of the puppet master
+        :type source: str
+        :param deploy_key: Deploy key to link with your puppet master
+        :type deploy_key: ppaas.DeployKey
+        :param nb: Number of servers for the master
+        :type nb: int
+        :param type: Kind of server to use
+        :type type: str
+        :param hierarchy: Array of hierarchies
```
Comment (thomas-maurice) [inline ppaas/master.py:123]: It could be good to create the Hierarchy objects too

File: ppaas/master.py [hunk 2]
```diff
@@ -107,6 +107,48 @@ def reload_data(self, refreshed_datas=None):
...
+        :type hierarchy: list
+        :param hieras: Array of hiera backend
```
Comment (thomas-maurice) [inline ppaas/master.py:125]: Same

**Comment by thomas-maurice** [comment]
The overall code seems good, will need some rework when the missing objects will be implemented, gg !

**Comment by rbeuque74** [comment]
Didn't know how this all works.
We should go over documentation on live example of master that have hierarchies and hieras

**Comment by thomas-maurice** [comment]
Same, I have to search about it a bit

**Pull request merged**